extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
@honeybadgerdontcare The change as recommended still doesn't quite work. It looks like the validator sees 

```html
<amp-state id="myState2" src="https://www.google.com/" credentials="omit"></amp-state>
```

and believes it's the `<amp-state>` with child tag. Any recommendation to get around this issue? If not I can try to combine the rules again and check at runtime that at least one state source is provided.